tasks/Debian.yml

```diff
@@ -3,7 +3,13 @@
   apt_key: id=E56151BF keyserver=keyserver.ubuntu.com state=present
 
 - name: Add Mesosphere repo
-  apt_repository: repo="{{marathon_apt_repo}}" state=present update_cache=yes
+  apt_repository: repo='deb {{ marathon_apt_url }} {{ ansible_distribution_release | lower }} main' state=present update_cache=yes
```

Comment:
use `marathon_os_distribution` instead of `ansible_distribution_release | lower`